arenadata/Dockerfile.ubuntu

```
@@ -39,7 +39,7 @@ RUN mkdir /home/gpadmin/bin_gpdb
 
 ENV TARGET_OS=ubuntu
 ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
-ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
+ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend CFLAGS=-DPy_UNICODE_WIDE"
```

Can this potentially break anything else? Can we (re)install a proper `subprocess32` module that uses the correct API? Did CentOS have the same option enabled, why did it work there?

Also, according to python2.7 sources, the define probably should look like this:
```suggestion
ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend CFLAGS=-DPy_UNICODE_SIZE=4"
```

---

This problem is cased when we install both of python (a and 3). CentOS does not have python3 (installed).

---

Why?

---

Centos will have the same problem, if python3 is installed into it. Lib `_posixsubprocess` is made with our GP, and it has not any flags to make it with Unicode 4 bytes. By default it is compiled with Unicode 4 bytes. Sources of python has file `unicodeobject.h`, which has lines:
```c
#ifndef Py_UNICODE_WIDE
...
# define PyUnicode_EncodeUTF8 PyUnicodeUCS2_EncodeUTF8
...
#else
...
# define PyUnicode_EncodeUTF8 PyUnicodeUCS4_EncodeUTF8
...
#endif
```
We can only set flag before making GP.